CS/DataContext_Ex/MainWindow.xaml

```diff
@@ -1,96 +1,117 @@
+<Window
+    x:Class="DataContext_Ex.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:dxd="http://schemas.devexpress.com/winfx/2008/xaml/docking"
+    xmlns:local="clr-namespace:DataContext_Ex"
+    Title="MainWindow"
+    Width="525"
+    Height="449">
     <dxd:DockLayoutManager x:Name="dm1">
-        <!--Set DataContext for DockLayoutManager. By default, this DataContext will be inherited by all child elements-->
+        <!--  Set DataContext for DockLayoutManager. By default, this DataContext will be inherited by all child elements  -->
         <dxd:DockLayoutManager.DataContext>
             <local:MyDescriptionClass Description="This example demonstrates how to bind layout elements to binding sources" />
         </dxd:DockLayoutManager.DataContext>
-
         <dxd:LayoutGroup Orientation="Vertical">
-            <dxd:LayoutPanel x:Name="panelDescription" Caption="Description" ItemHeight="70">
+            <dxd:LayoutPanel
+                x:Name="panelDescription"
+                Caption="Description"
+                ItemHeight="70">
                 <dxd:LayoutGroup Orientation="Vertical">
-                    <!--Define a label and bind its caption to MyDescriptionClass.Description.
-                    A MyDescriptionClass object is assigned to the DockLayoutManager.DataContext, and this DataContext is inherited by all layout items.
-                    Since the label is not a FrameworkElement, the binding statement must be created using the explicit syntax.
+                    <!--
+                        Define a label and bind its caption to MyDescriptionClass.Description.
+                        A MyDescriptionClass object is assigned to the DockLayoutManager.DataContext, and this DataContext is inherited by all layout items.
+                        Since the label is not a FrameworkElement, the binding statement must be created using the explicit syntax.
                     -->
                     <dxd:LabelItem Caption="{Binding Path=DataContext.Description, RelativeSource={RelativeSource Mode=Self}}" />
                 </dxd:LayoutGroup>
             </dxd:LayoutPanel>
-
             <dxd:LayoutGroup Orientation="Horizontal">
-
-                <!--The panel that has its own DataContext-->
+                <!--  The panel that has its own DataContext  -->
                 <dxd:LayoutPanel x:Name="panelPerson1" Caption="Person 1">
-                    <!--Provide DataContext for this panel and its child controls-->
+                    <!--  Provide DataContext for this panel and its child controls  -->

```

Comment:
Please update this comment without using "Provide".

Reply:
```suggestion
                    <!--  Define DataContext for this panel and its child controls  -->

```